src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,103 @@
 package mate.jdbc;
 
+import java.util.List;
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.CarService;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        Driver firstDriver = new Driver();
+        firstDriver.setLicenseNumber("first");
+        firstDriver.setName("Ivan");
+        firstDriver.setId(15L);
```

Comment:
let's not do this
let id be autogenerated and then use `firstDriver.getId()` after you save it to db

same with all other entities